tests/Shiny.Tests/Locations/PositionTests.cs

```diff
@@ -36,5 +41,45 @@ public void Outside_Geofence()
                 .Should()
                 .Be(false);
         }
+
+
+        [Theory]
+        [InlineData(lat0, lng0, lat0, lng0, true)]
+        [InlineData(lat0, lng0, lat1, lng0, false)]
+        [InlineData(lat0, lng0, lat0, lng1, false)]
+        [InlineData(lat0, lng0, lat1, lng1, false)]
+        public void EqualsPosition(double lat1, double lng1, double lat2, double lng2, bool expected) =>
+            new Position(lat1, lng1)
+                .Equals(new Position(lat2, lng2))
+                .Should()
+                .Be(expected);
+
+
+        [Fact]
+        public void EqualsNull() =>
+            new Position(lat0, lng0)
+                .Equals(null)
+                .Should()
+                .Be(false);
+
+
+        [Theory]
+        [InlineData(lat0, lng0, lat0, lng0, true)]
+        [InlineData(lat0, lng0, lat1, lng0, false)]
+        [InlineData(lat0, lng0, lat0, lng1, false)]
+        public void GetHashCodeConsidersLatitudeAndLongitude(double lat1, double lng1, double lat2, double lng2, bool expected) =>
+            new Position(lat1, lng1)
+                .GetHashCode()
+                .Equals(new Position(lat2, lng2).GetHashCode())
+                .Should()
+                .Be(expected);
+
+
+        [Fact]
+        public void ToStringReturnsLatitudeAndLongitude() =>
+            new Position(lat0, lng0)
+                .ToString()
+                .Should()
+                .Be($"Latitude: {lat0} - Longitude: {lng0}");
```

Comment:
I considered changing this to `$"[Position: {lat0},{lng0}]"` to be more similar to the others, but figured I'd start with the passing test.